bip-full-rbf-deadline.mediawiki

```diff
@@ -0,0 +1,97 @@
+<pre>
+  BIP: ??
+  Title: Full Replace-by-Fee Deployment Schedule
+  Author: Peter Todd <pete@petertodd.org>
+  Status: Draft
+  Type: Standards Track
+  Created: 2015-06-29
+</pre>
+
+==Summary==
+
+This BIP proposes a deployment schedule for full replace-by-fee (full-RBF)
+functionality, with an automatic activation of Tuesday April 5th, 2016, at 3pm
+UTC upon which supporting relay nodes and miners will enable full-RBF mempool
+behavior on mainnet. Prior to the activation deadline supporting nodes and
+miners will support first-seen-safe(1) replace-by-fee (FSS-RBF) mempool behavior.
+
+
+==Motivation==
+
+Full-RBF has significant efficiency advantages(2) over alternatives such as
+FSS-RBF and Child-Pays-For-Parent for a wide variety of common transaction
+patterns such as fee-bumping and multiple sequential payments, as well as smart
+contract protocols such as payment channels and auctions. Miner support would
+let the wider Bitcoin community use the blockchain more efficiently, supporting
+more transactions per second in less blockchain space.
+
+While full-RBF does allow users to "undo" transactions after they have been
+sent, the ability of decentralized wallets to protect users from double-spends
+has proven to be near-zero.(3) Centralized services have had some success in
```

Comment:
Elaborate on how it is proven to be near-zero. This is an important point that deserves its own section imho, just just a reference to a mail (you sent, by the way) on the mailing list.

Even copy-pasting the contents of your mail + formatting would be much better.